src/views/Registration/RegistrationDefineBusiness.vue

```diff
@@ -179,6 +194,7 @@ export default class RegistrationDefineBusiness extends Mixins(CommonMixin) {
   /** Object of valid flags. Must match validComponents. */
   get validFlags (): object {
     return {
+      businessTypeConfirmValid: this.isTypePartnership ? this.businessTypeConfirmValid : true,
```

Comment:
You need to update this for validation to work for SP businesses.

Also, if there are any comments that say Business Type Confirm is for GP only, please remove them (and possibly remove the optional designator) (and possibly update the initialization and fallback in filing-template-mixin to False since that should be the initial value for both SP and GP).

Reply:
Fixed